e2e/config.go

```diff
@@ -37,16 +37,21 @@ func (c hclConfig) appendPort(port int) hclConfig {
 	return c.appendString(fmt.Sprintf("port = %d", port))
 }
 
-func (c hclConfig) appendConsulBlock(consul *testutil.TestServer) hclConfig {
+func (c hclConfig) appendConsulBlock(consul *testutil.TestServer, opts ...string) hclConfig {
+	var optsConfig string
+	if len(opts) > 0 {
+		optsConfig = "\n" + strings.Join(opts, "\n")
```

Comment:
You add a newline when you do the `Sprintf` so I don't think you need the first `"\n"`

Makes it slightly easier to read